cypress/integration/e2e/opengraph/index.spec.ts

```diff
@@ -0,0 +1,17 @@
+describe('OpenGraph Cards', () => {
+  it('renders opengraph cards correctly', () => {
+    cy.visit('/opengraph/?path=components/alert');
```

Comment:
Hmm is it enough to just test that `alert` works? Just thinking about future components that get added and might not pass `getHumanizedNameFromPackageName()` or get added to `ManualPackageNameMapping`. Not sure what other edge cases might make the card creation fail, but would it be too time-consuming to check that each page works here? I don't think we'd have to test each of these things on each page (h1, svg, area label, font), but just that they work.

I wonder if there's any way to add the opengraph urls to the links that the broken link checker checks?

Reply:
I added two VRT stories for Alert and Button to test the visual rendering works as intended, with every possible option for the statuses on top of this Cypress test.  Since this is one endpoint for every component, I believe testing just one endpoint would be alright as all the data is retrieved in the same way.

Reply:
Makes sense ✔️